src/components/AssetSearch/stories/AssetSearch.stories.tsx

```diff
@@ -191,4 +192,41 @@ storiesOf('AssetSearch/Examples', module)
         content: customStyles,
       },
     }
+  )
+  .add(
+    'Handle Search Results',
```

Comment:
Change to `Handle search results` to look similar to other story names

Reply:
fixed